### PR TITLE
Modify Pprent/child relationship FF for Dynamically Updating Results in SSCS

### PR DESCRIPTION
**Pull request by stephherbers**
## Product Description

Makes modifications for [this ticket](https://dimagi-dev.atlassian.net/browse/USH-3544) so that the new FF for dynamically updating search results that is a child of split screen chase search

## Technical Summary

To handle the domains where the SSCS FF is already enabled and this new FF will therefore not be automatically enabled, this PR includes a management command. 

## Feature Flag

DYNAMICALLY_UPDATE_SEARCH_RESULTS

## Safety Assurance
The management command will be run locally and on staging prior to prod. Only the status of the feature flag is being modified for a ff that does not control anything quite yet

### Automated test coverage

None

### QA Plan

I'm not requesting QA for this segment of the feature

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/toggles/__init__.py [hunk 1]
```diff
@@ -1030,14 +1030,23 @@ def _ensure_valid_randomness(randomness):
...
+
 SPLIT_SCREEN_CASE_SEARCH = StaticToggle(
     'split_screen_case_search',
     "Split screen case search: In case search, show the search filters in a sidebar on the left and the results"
     " on the right.",
     TAG_CUSTOM,
     help_link='https://confluence.dimagi.com/display/USH/Split+Screen+Case+Search',
     namespaces=[NAMESPACE_DOMAIN],
-    parent_toggles=[SYNC_SEARCH_CASE_CLAIM]
+    parent_toggles=[SYNC_SEARCH_CASE_CLAIM, DYNAMICALLY_UPDATE_SEARCH_RESULTS]
```
Comment (nospame) [inline corehq/toggles/__init__.py:1049]: Believe Simon commented on the other PR, but I think this is backwards, right? `DYNAMICALLY_UPDATE_SEARCH_RESULTS` would probably require `SPLIT_SCREEN_CASE_SEARCH` to be enabled, not the other way around.

**Review by nospame — COMMENTED** [review]
Just the one blocking comment, otherwise looks good.

File: corehq/toggles/__init__.py (hunk 1 above)
Comment (stephherbers) [reply corehq/toggles/__init__.py]: resolved, and also [see comment here](https://github.com/dimagi/commcare-hq/pull/33429#issuecomment-1711711066)

**Review by MartinRiese — COMMENTED** [review]
I know it is pretty unlikely that self hosted instances are using this feature flag but should we not add an actual migration for this too?

**Review by nospame — APPROVED** [review]

**Comment by stephherbers** [comment]
> I know it is pretty unlikely that self hosted instances are using this feature flag but should we not add an actual migration for this too?
I thought about this when opting for the one off command given that there are no model changes and we really only are about prod for this migration-- otherwise the flag can be manually enabled

**Comment by stephherbers** [comment]
no longer needed with changes combined with https://github.com/dimagi/commcare-hq/pull/33429

**Review by orangejenny — APPROVED** [review]

**Pull request merged**